build/gen_stub.php

```diff
@@ -3003,8 +3097,8 @@ public function getDeclaration(array $allConstInfos): string {
             $code .= $defaultValue->initializeZval($zvalName);
         }
 
-        $code .= "\tzend_string *property_{$propertyName}_name = zend_string_init(\"$propertyName\", sizeof(\"$propertyName\") - 1, 1);\n";
-        $nameCode = "property_{$propertyName}_name";
+        [ $stringInit, $nameCode, $stringRelease ] = $this->getString( $propertyName );
```

Comment:
I usually try to keep away from commenting coding style, but this one (spaces before and after brackets) is completely off from gen_stub.php.

Reply:
No problem, done